yamlExamples/1/example.yaml

```diff
@@ -0,0 +1,42 @@
+# same as defined in current version
+agent:
+  label: my_label
+  customWorkspace: path_to_workspace
+
+tools:
+  tool_1 : tool_1_name
+  tool_2 : tool_2_name
```

Comment:
I think these examples will be more useful if they're real - what tool would be installed

This comment applies to the full PR